NodeJS/src/AgentSkill.ts

```diff
@@ -0,0 +1,47 @@
+const args = require('minimist')(process.argv.slice(2));
+export default args
+
+const ENDPOINT_KEY = 'endpoint';
+const ADMIN_USERNAME_KEY = 'admin_username';
+const ADMIN_PASSWORD_KEY = 'admin_password';
+const REPLACE_REGEX = /'/g;
+const EMPTY_STRING = "";
+let endpoint = args[ENDPOINT_KEY].replace(REPLACE_REGEX, EMPTY_STRING);
```

Comment:
what if args[ENDPOINT_KEY] is null or undefined?

Reply:
handled in function **isValidParameter**.

You can just check if the variable has a truthy value or not. That means

if( value ) {
}
will evaluate to true if value is not:

null
undefined
NaN
empty string ("")
0
false